hawc/apps/lit/managers.py

```diff
@@ -100,9 +100,11 @@ def get_from_ris(self, search_id, references):
             ids.append(ident)
 
             # create DOI identifier
-            if ref["doi"] is not None:
+            if constants.DOI_EXTRACT.search(str(ref["doi"])):
```

Comment:
Make a variable out of this regex search so that you don't have to perform the same search below.

Reply:
Do this for the other manager methods too.